blocks/editable/index.js

```diff
@@ -564,6 +595,10 @@ export default class Editable extends Component {
 				} else {
 					this.editor.execCommand( 'Unlink' );
 				}
+			} else if ( format === 'footnote' ) {
+				const footnoteId = uuid();
+				this.footnotes[ footnoteId ] = formatValue.text;
+				this.editor.selection.setContent( `<a href="#footnote-${ footnoteId }" data-footnote-id="${ footnoteId }" contenteditable="false"><sup>[*]</sup></a>` );
```

Comment:
This is being persisted into the saved content of the post, which is problematic because we don't want/care for this editor-specific attribute to be present on the front-end of the site. Also, it's causing React to log a warning into the developer console.